pom.xml

```diff
@@ -51,6 +51,7 @@
             <scope>test</scope>
             <type>jar</type>
         </dependency>
+

```

Comment:
Might want to clean up some of the extra spacing/newlines here?  Unless you have it there for a reason?